packages/clarity-js/src/data/metadata.ts

```diff
@@ -149,9 +150,11 @@ function track(u: User, consent: BooleanFlag = null): void {
   // Convert time precision into days to reduce number of bytes we have to write in a cookie
   // E.g. Math.ceil(1628735962643 / (24*60*60*1000)) => 18852 (days) => ejo in base36 (13 bytes => 3 bytes)
   let end = Math.ceil((Date.now() + (Setting.Expire * Time.Day))/Time.Day);
+  // If DOB is not set in the user object, use today's date as a DOB
+  let dob = u.dob === null ? new Date().toISOString().substring(0, 10).replace(/-/g,'') : u.dob;
```

Comment:
Couple of points here: 
- This will convert the date to UTC and might not align with server date: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
- Since these bytes will be added to every cookie we should consider compression. The easiest it to turn into base36 but we can also subtract it from a pivot date and then base36 it (e.g. current date - 20200101) and only send the delta.

Reply:
I have created a PR for ingest, could you take a look and review this one again please.